python_src/src/lamp_py/tableau/pipeline.py

```diff
@@ -25,6 +27,16 @@
 
 def start_hyper_updates() -> None:
     """Run all HyperFile Update Jobs"""
+    validate_environment(
+        required_variables=[
+            "TABLEAU_USER",
+            "TABLEAU_PASSWORD",
+            "TABLEAU_SERVER",
+            "PUBLIC_ARCHIVE_BUCKET",
+        ],
+        validate_db=True,
```

Comment:
this app doesn't need to connect to the DB does it? its reading from s3, converting, and publishing.

Reply:
ah yes, `run_hyper` functions don't require DB access, only `run_parquet`, will fix that.